satpy/etc/composites/msi.yaml

```diff
@@ -1,72 +1,137 @@
 sensor_name: visir/msi
 
-
 modifiers:
-
-  rayleigh_corrected:
+  rayleigh_corr:
     modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
     atmosphere: us-standard
     aerosol_type: rayleigh_only
     prerequisites:
-    - name: 'B04'
-      modifiers: [sunz_corrected]
-    optional_prerequisites:
-    - satellite_azimuth_angle
-    - satellite_zenith_angle
-    - solar_azimuth_angle
-    - solar_zenith_angle
-
-  rayleigh_corrected_marine_clean:
+      - name: 'B04'
+        modifiers: [effective_solar_pathlength_corrected]
+      - name: satellite_azimuth_angle
+      - name: satellite_zenith_angle
+      - name: solar_azimuth_angle
+      - name: solar_zenith_angle
```

Comment:
why are the angles not optional anymore?

Reply:
I'll just restore it but I don't know if you can do the correction without these angle data...That's why I make them necessary.